src/algorithm/broadcasting/primitive.rs

```diff
@@ -49,3 +49,15 @@ where
         }
     }
 }
+
+impl From<f64> for BroadcastablePrimitive<Float64Type> {
+    fn from(value: f64) -> Self {
+        BroadcastablePrimitive::Scalar(value)
+    }
+}
+
+impl From<u32> for BroadcastablePrimitive<UInt32Type> {
+    fn from(value: u32) -> Self {
+        BroadcastablePrimitive::Scalar(value)
+    }
+}
```

Comment:
I wonder if it's possible to make a blanket implementation for anything that implements `ArrowPrimitiveType`, so you don't have to make a separate copy for `f64`, `f32`, `u8`, etc.

Reply:
Yeah, this is cleaner.